test/functional/services/doc_table.js

```diff
@@ -54,7 +55,12 @@ export function DocTableProvider({ getService }) {
 
     async toggleRowExpanded(row) {
       const rowExpandToggle = await this.getRowExpandToggle(row);
-      return await rowExpandToggle.click();
+      await rowExpandToggle.click();
+
+      const detailsRow = await row.findByXpath('./following-sibling::*[@data-test-subj="docTableDetailsRow"]');
+      return await retry.try(async () => {
+        return detailsRow.findByCssSelector('[data-test-subj~=docViewer]');
```

Comment:
Don't [string values](https://www.w3.org/TR/CSS21/syndata.html#strings) have to be quoted in css? Not sure why the unquoted `docViewer` works.

Reply:
Huh, weird. Mistake on my part in any case, went ahead and added the double quotes just in case.